<?xml version="1.0"?>
<launch>
  <arg name="object_tf_file" default="scene_1.yaml" />

  <include file="$(find photoneo_localization)/launch/localization_node.launch">
    <arg name="scene_source" value="PhoXiTemp(0)(File3DCamera)" />
    <arg name="reference_frame" value="photoneo_center_optical_frame" />
    <arg name="target_names_file" value="target_names_HV8_jigHV6.yaml" />
  </include>

  <node pkg="photoneo_localization" type="tf_broadcaster.py" name="initial_object_tf_broadcaster" output="screen" />

  <node pkg="teaching_processing" type="save_tf_pose.py" name="save_object_tf_pose_node" output="screen">
    <param name="object_tf_file" value="$(arg object_tf_file)" />
  </node>
</launch>
